pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -5614,3 +5614,21 @@ MULTILINESTRING ((0 0, 1 1), (1 1, 2.5 2.5), (1 3, 2.5 2.5), (2.5 2.5, 4 4), (4
 
 statement error geometry type is unsupported. Please pass a LineString or a MultiLineString
 SELECT ST_AsText(ST_Node(ST_GeomFromText('POLYGON((1 1, 1 3, 3 3, 3 1, 1 1))')));
+
+
+subtest st_subdivide
```

Comment:
you can ignore this word :)

Reply:
Removed

Reply:
But out of curiosity - is not this `subtest` helpful? 🤔

Reply:
trying to remove all usages of it (coming soon) -- the problem is people keep adding to the end of the file and the meaning is kind of lost.